src/typeDefs.ts

```diff
@@ -11,6 +11,7 @@ export const typeDefs = gql`
     description: String
     quantity: Int!
     notes: String
+    tags: [String!]
```

Comment:
I don't really understand the syntax, but I've been preferring...

```suggestion
    tags: [String!]!
```

Reply:
I think what I had means "you _might_ get a list of tags but if you do they will all be non-null" whereas yours means "you _will_ get a list and they will all be non-null" (although it could be an empty list)